src/main/java/chess/database/Connector.java

```diff
@@ -0,0 +1,35 @@
+package chess.database;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class Connector {
+    private static final String SERVER = "localhost:13306";
+    private static final String DATABASE = "db_name";
+    private static final String OPTION = "?useSSL=false&serverTimezone=UTC";
+    private static final String USERNAME = "root";
+    private static final String PASSWORD = "root";
+    private static final String URL = String.format("jdbc:mysql://%s/%s%s", SERVER, DATABASE, OPTION);
+
+    public static Connection getConnection() {
```

Comment:
https://github.com/woowacourse/java-chess/pull/116#discussion_r405875063 피드백 의미 전달에 문제가 있던 것 같네요 T.T

먄약 해당 메서드에서 Connection이 null로 응답되는 것이 의도됐다면 

```suggestion
    public static Optional<Connection> getConnection() {
```

와 같은 시그니쳐를 가지면 좋고, Null이 오는 상황이 일반적이지 않다면 

```java
        try {
            return DriverManager.getConnection(URL, USERNAME, PASSWORD);
        } catch (SQLException e) {
            // TODO throw
        }
```

와 같은 코드로 충분하지 않을까 했습니다!